config/version.txt

```diff
@@ -1 +1 @@
-dev
```

Comment:
?

Reply:
This must've happened automatically? will remove